### PR TITLE
Homework3: Alexander Zhuk

### PR DESCRIPTION
**Pull request by ZhukAlexander90**

### CODE REVIEW

File: Homework3_ZhukAlexander/task1_fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Печатает цифры от 1 до 100
+# вместо кратных 3 пишет FIzz
+# вместо кратных 5 пишет Buzz
+# вместо одновременно кратных 3 и 5 пишет
+# FizzBuzz
+
+for num in range(1, 101):
+    if not num % 3 and not num % 5:
```
Comment (belovmd) [inline Homework3_ZhukAlexander/task1_fizzbuzz.py:8]: 15?

File: Homework3_ZhukAlexander/task2_list_practice.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# Одной строкой удалите элемент  '2a' из прошлого
+# списка и напечатайте его.
+# Скопируйте список и добавьте в него элемент '2a' так
+# чтобы в исходном списке этого элемента не было.
+
+import copy
+first_lst = [first + second for first in "ab" for second in "bcd"]
+print(first_lst)
+print(first_lst[::2])
+second_lst = [str(num) + letter for num in range(1, 5) for letter in "a"]
```
Comment (belovmd) [inline Homework3_ZhukAlexander/task2_list_practice.py]: '1234' ?

**Review by belovmd — COMMENTED** [review]
Please address these issues

File: Homework3_ZhukAlexander/task1_fizzbuzz.py (hunk 1 above)
Comment (ZhukAlexander90) [reply Homework3_ZhukAlexander/task1_fizzbuzz.py:8]: 15 multiple at the same time 3 and 5
15 % 3 == 0
15 % 5 == 0

File: Homework3_ZhukAlexander/task2_list_practice.py (hunk 2 above)
Comment (ZhukAlexander90) [reply Homework3_ZhukAlexander/task2_list_practice.py]: done

File: Homework3_ZhukAlexander/task1_fizzbuzz.py (hunk 1 above)
Comment (belovmd) [reply Homework3_ZhukAlexander/task1_fizzbuzz.py:8]: if not num % 15 in first conditions much more explicit and do the same

**Review by belovmd — COMMENTED** [review]
9

**Pull request closed without merging**